openc3-cosmos-cmd-tlm-api/Dockerfile

```diff
@@ -16,11 +16,10 @@ RUN bundle config set --local without 'development' \
     /var/tmp/*
 
 RUN ["chown", "-R", "openc3:openc3", "/src/"]
```

Comment:
Is this needed if we're just going to chmod 777 in two lines?

Reply:
It still sets the expected owner.